index.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const app = express();
+const port = process.env.PORT || 4000;
```

Comment:
all imports at the top and there is an order for best practice:
1. libraries and modules
2. your files (routers, models) grouped by concern (same in the front end with components/actions)
3. css files (front end)